microsite/package.json

```diff
@@ -32,6 +32,9 @@
     "@docusaurus/core": "2.4.0",
     "@docusaurus/plugin-client-redirects": "^2.3.1",
     "@docusaurus/preset-classic": "2.4.0",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@mui/material": "^5.12.1",
```

Comment:
I would contest that adding the entire MUI and emotion here, is overkill. We've been holding off on doing that intentionally. The infima package that comes builtin with docusaurus, [has pills](https://infima.dev/docs/components/pills/) that we probably should use instead.

Reply:
Thanks for the feedback, that make sense! I missed the infima integration, will make the changes and remove the material dependencies!